src/views/properties/properties.js

```diff
@@ -4,12 +4,22 @@ import UserContext from '../../UserContext';
 import { Link } from "react-router-dom"
 import * as axios from 'axios';
 import Search from '../../components/Search';
+import { ShowHideSwitch } from '../../components/ShowHideSwitch';
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faArchive } from '@fortawesome/free-solid-svg-icons';
 import Toast from '../../utils/toast';
 
 import './properties.scss'
 
 const columns = [{
   dataField: 'name',
+  formatter: (_, row) => {
+    return (
+      <Link key={row.id} to={`/manage/property/${row.id}`}>
```

Comment:
The selectRow functionality seems to be overriding this Link functionality as I can't click into the property details page using the property name

Reply:
This part is going to be weird until we have a view/edit properties page

Reply:
We have a view/edit properties page - hooray! Thank you @Protopaco!
Clicking on the property name now works as expected. Please take a look @Chris-Boe